components/automate-chef-io/content/docs/iam-v2-api-reference.md

```diff
@@ -74,227 +157,548 @@ You can see the actions contained in the `editor` role in the [Roles section]({{
+create | POST /projects            | creates a project
+update | PUT /projects/***id***    | updates a project
+delete | DELETE /projects/***id*** | deletes a project
+
+**Example Project:**
+
+```json
+{
+  "name": "Test Project",
+  "id": "test-project-1"
+}
 ```
 
-In the browser: Navigate to the *Roles* page under the **Settings** tab, and then select an individual role to open its detail view.
+### Listing Projects
 
-### Creating a Role
+The output lists all defined projects.
 
-In the IAM v2 beta, custom roles must be created on the command line.
-Use the following `curl` command to create a role combining some editor actions with some administrator actions:
+*In the browser:* **Settings**  >> **Projects**
 
-```bash
-curl -s -H "api-token: $TOKEN" -d '{"name": "Test Role", "id": "test-role-1", "actions": ["infra:*", "compliance:*", "teams:*", "users:*"] }' https://{{< example_fqdn "automate" >}}/apis/iam/v2beta/roles?pretty
+### Getting a Project
+
+This shows the details of a single project, selected by its ID.
+
+*In the browser:* **Settings**  >> **Projects** >> [select a project]
+
+### Creating a Project
+
+Create a project by composing JSON with all the necessary project properties (see the beginning of the section above).
+
+### Updating a Project
+
+- The project name is the only property of a project you can modify.
+- The ID is immutable; it can only be set at creation time.
+
+### Deleting a Project
+
+Deleting a project is permanent and cannot be undone.
+
+*In the browser:* **Settings**  >> **Projects** >> [open control menu for target project] >> Delete Project
+
+## Project Rules
+
+Method | HTTP request                                          | Description
+-------|-------------------------------------------------------|------------
+list   | GET /projects/***project_id***/rules                  | lists rules for a project
+get    | GET /projects/***project_id***/rules/***rule_id***    | gets a project rule
+create | POST /projects/***project_id***/rules                 | creates a project rule
+update | PUT /projects/***project_id***/rules/***rule_id***    | updates a project rule
+delete | DELETE /projects/***project_id***/rules/***rule_id*** | deletes a project rule
+
+A rule specifies a non-empty array of conditions.
+Each rule **condition** specifies values to match for a particular **attribute**.
+The choice of attributes depends on the rule **type**.
+
+Rule Type  | Available Attributes
+-----------|---------------------
+EVENT      | CHEF_ORGANIZATION, CHEF_SERVER
+NODE       | CHEF_ORGANIZATION, CHEF_SERVER, ENVIRONMENT, CHEF_ROLE, CHEF_TAG, CHEF_POLICY_NAME, CHEF_POLICY_GROUP
+
+Each rule condition also specifies an **operator**, either `MEMBER_OF` or `EQUALS`.
+To match a single value, use `EQUALS` and include just a single value in the **values** array.
+To match any one of a set of values, use `MEMBER_OF` and include the choices in the **values** array.
```

Comment:
This might be a good spot for this Important Detail: 

> Emphasize that all rule conditions must be true for the rule to apply. (for example: the conditions chef_org EQUALS project1 and chef_org EQUALS project2 contradict each other. the operator MEMBER_OF should be used instead)